authui/src/authflowv2/components/text-field.ts

```diff
@@ -0,0 +1,30 @@
+import { Controller } from "@hotwired/stimulus";
+
+export class TextFieldController extends Controller {
+  static values = {
+    inputErrorClass: { type: String, default: "input--error" },
+  };
+  static targets = ["input", "errorMessage"];
+
+  declare inputErrorClassValue: string;
+  declare errorMessageTarget: HTMLElement | null;
+  declare inputTarget: HTMLInputElement;
+
+  connect() {
+    this.inputTarget.addEventListener("input", this.onInput);
+  }
+
+  disconnect() {
+    this.inputTarget.removeEventListener("input", this.onInput);
+  }
+
+  onInput = () => {
+    if (this.inputTarget.classList.contains(this.inputErrorClassValue)) {
+      this.inputTarget.classList.remove(this.inputErrorClassValue);
+    }
+
+    if (this.errorMessageTarget != null) {
+      this.errorMessageTarget.style.display = "none";
```

Comment:
Set the tailwind class `hidden` to hide the element.

Reply:
ok. But if we use tailwind css class in typescript files, I think we need to include the file paths in authui/tailwind.config.js so that tailwind won't remove the classes used in script.

Reply:
I meant using the literal `"hidden"` only. I guess it is available by default. No need to mess with typescript.

Reply:
Tailwind removes all classes that are unused from the generated stylesheet, so you need to tell it where to find those classes:
https://tailwindcss.com/docs/content-configuration#class-detection-in-depth

I think I just add `"./src/**/*.{css,scss,ts}"` to the tailwind config.

Reply:
I see. Though `hidden` is used in many places already.